tox.ini

```diff
@@ -1,17 +1,41 @@
 [tox]
 envlist =
-    {py27,py33,py34,py35}-django18
-    {py27,py34,py35}-django19
-    {py27,py34,py35}-django110
-    {py27,py34,py35,py36}-django111
+    lint
+    py{27,33,34,35}-django18
+    py{27,34,35}-django{19,110}
+    py{27,34,35,36}-django111
+
+[travis]
+python =
+  2.7: py27
+  3.3: py33
+  3.4: py34
+  3.5: py35
+  3.6: py36, coverage, lint
```

Comment:
This is the key trick! When you run travis it runs the environments one by one. But we only bother with coverage and linting if you're using python 3.6.